index.d.ts

```diff
@@ -1,3 +1,16 @@
+interface Options {
+	/**
+	Optional filter function. function should return false to prevent the module from being removed from cache.
```

Comment:
Text formatting.

Reply:
Unknown what to do, there are no defined rules on text formatting, this should be documented or linting rules should be created on what the format should entail